### PR TITLE
(improvement): Move unit tests to github actions

### PR DESCRIPTION
**Pull request by PeterNgTr**
## Motivation/Description of the PR
This PR aims to move unit tests to github actions to separate from matrix. While it costs some times to run again same unit tests in every helper.

**Before** every time travis ci is kicked off, the unit tests are run for each helper (5 helpers as of now) before helper tests. 

**After** the unit tests are kicked off to run in parallel with helper tests and run once :)

https://github.com/Codeception/CodeceptJS/pull/2138/checks?check_run_id=390693242

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Unit Tests
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```
Comment (pablopaul) [inline .github/workflows/nodejs.yml]: @PeterNgTr I suggest to use a fixed ubuntu version here

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: run unit tests
+      run: |
+        npm i
+        npm test
+      env:
+        CI: true
```
Comment (pablopaul) [inline .github/workflows/nodejs.yml]: @PeterNgTr isnt that env var set by default? Most CI systems do set it by default, but I am unsure about GitHub.

File: .github/workflows/nodejs.yml (hunk 1 above)
Comment (PeterNgTr) [reply .github/workflows/nodejs.yml]: it is by default of github actions and being used here -> https://github.com/Codeception/CodeceptJS/blob/master/.github/workflows/check.yml#L5

do you have suggestion which version should it be?

File: .github/workflows/nodejs.yml (hunk 2 above)
Comment (PeterNgTr) [reply .github/workflows/nodejs.yml]: yeah, it's be default. I could try to remove it to see if anything is impacted.

Comment (PeterNgTr) [reply .github/workflows/nodejs.yml]: okay, nothing is impacted!

File: .github/workflows/nodejs.yml (hunk 1 above)
Comment (pablopaul) [reply .github/workflows/nodejs.yml]: @PeterNgTr I suggest to use current latest explicitely, it would be `ubuntu-18.04`

File: .travis.yml [hunk 3]
```diff
@@ -26,8 +26,8 @@ before_script:
 - docker run -d --net=host --shm-size=2g selenium/standalone-chrome:3.141.59-oxygen
 - export DBUS_SESSION_BUS_ADDRESS=/dev/null
 - export DISPLAY=:99.0
+- sleep 3
```
Comment (pablopaul) [inline .travis.yml]: @PeterNgTr for what to we wait here?

Comment (PeterNgTr) [reply .travis.yml]: wait for selenium is ready otherwise it will be failed.

**Comment by DavertMik** [comment]
```
/home/runner/work/CodeceptJS/CodeceptJS/node_modules/runio.js/src/command.js:5
```

It seems like a problem in `runio`. I suspect this is due to dependency of `parse-function` they changed something again.

**Comment by DavertMik** [comment]
@PeterNgTr I updated runio to version 1.0.20 and this should fix an issue

**Comment by PeterNgTr** [comment]
> @PeterNgTr I updated runio to version 1.0.20 and this should fix an issue

Thanks @DavertMik I updated it here.

**Comment by elukoyanov** [comment]
@DavertMik , link to skipped test is wrong. Testomatio problems?

> 🌀 Tests overview by [Testomatio](https://testomat.io)
> 
> Found **839** mocha tests in 69 files
> No new tests added or removed
> 
> ⚠️ List all skipped tests (1)
> * [~should resize window to maximum screen dimensions~](https://github.com/Codeception/CodeceptJS/tree/97bb78452befdf6a2d16f3c5fa4fde25d39eb238/test/helper/WebDriver_test.js#L610)
> 
> 📎 List all suites (56)

**Comment by elukoyanov** [comment]
@PeterNgTr I don't see removing of unit tests from travis (any changes in travis). It's silly, but I don't see they run or not.

> **Before** every time travis ci is kicked off, the unit tests are run for each helper

**Comment by DavertMik** [comment]
@elukoyanov right, thanks for reporting!

File: .github/workflows/nodejs.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      matrix:
+        node-version: [10.x, 12.x]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: run unit tests
```
Comment (DavertMik) [inline .github/workflows/nodejs.yml]: please do not mix with check-tests, let's run them in parallel.
create a new file "tests.yml" here and run only unit tests

**Comment by elukoyanov** [comment]
@PeterNgTr , did you do all requested changes?

**Comment by PeterNgTr** [comment]
I think it's all done.

check-tests action is running in parallel with unittests action

Screenshot 2020-01-06 at 16 35 03

**Pull request merged**